NoteFind/Delegates/AppDelegate.h

```diff
@@ -6,11 +6,12 @@
 //
 
 #import <UIKit/UIKit.h>
-
+#import <CoreData/CoreData.h>
 
 @interface AppDelegate : UIResponder <UIApplicationDelegate>
+@property (nonatomic, strong) NSPersistentContainer *persistentContainer;
```

Comment:
you probably want this to also be `readonly` since you wouldn't want another part of the app to change the NSPersistentContainer created in the App Delegate

Reply:
done!